api/controllers/userController.js

```diff
@@ -0,0 +1,10 @@
+const { userService } = require('../services')
+const { catchAsync } = require('../utils/error')
+
+const kakaoSignIn = catchAsync(async(req, res) => {
+    const code = req.headers.authorization
+    const accessToken = await userService.kakaoSignIn(code)
+    res.status(200).json({accessToken:accessToken})
```

Comment:
```suggestion
    return res.status(200).json({accessToken})
```
응답뒤에 더 실행할 로직이 없다면 return 해주세요. 추가로 작성하신 코드처럼 오브젝트에서 key와 value가 일치한다면 다음과 같이 쓸 수 있습니다.